src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactoryTest.java

```diff
@@ -386,7 +398,8 @@ public void createWithSortOrderDescendingAndMixOfDateTypes() throws IOException,
                 CcdBundlePaginationStyle.off,
                 null,
                 null,
-                false
+                false,
+            null
```

Comment:
not a massive deal but do you want to have the null on same indents as the other vars here?